packages/sources/tradingeconomics-test/src/config/index.ts

```diff
@@ -0,0 +1,31 @@
+import { AdapterConfig } from '@chainlink/external-adapter-framework/config'
+
+export const defaultEndpoint = 'price'
+export const config = new AdapterConfig({
+  API_ENDPOINT: {
+    description: 'The HTTP URL to retrieve data from',
+    type: 'string',
+    default: 'https://api.tradingeconomics.com/markets',
+  },
+  WS_API_ENDPOINT: {
+    description: 'The WS URL to retrieve data from',
+    type: 'string',
+    default: 'wss://stream.tradingeconomics.com/',
+  },
+  API_CLIENT_KEY: {
+    description: 'The TradingEconomics API client key',
+    type: 'string',
+    required: true,
+  },
```

Comment:
Could you also mark this as `sensitive`? Based on how we have this configured in infra, it may be considered a secret too.